docs/pages/faq.mdx

```diff
@@ -63,6 +63,19 @@ are not logged in to Teleport or may not have access to a browser.
 Make sure that all Teleport components are at least at version 10.3.0. Older versions
 don't support the SFTP protocol, and it's enabled by default in `tsh` v11.0.0 and OpenSSH v9.0.
 
+## `tsh` is very slow on Windows, what to do?
+
+If your host machine is joined to one or more Active Directory domains, user lookups may take a long 
+time in your environment. To fix this, make sure `tsh` is at least at version 12.2.4. Either set
```

Comment:
It might be nice to add a sentence or two about why this happens and why this is the way to fix it. Something like this maybe (but note that I'm just making the following up as a "best guess"):
If your host machine is joined to one or more Active Directory domains, you might fine user lookups take a longer than you expect because of the number of Active Directory accounts that must be scanned in your environment. To fix this issue, you can use environment variables to preset account information for your Teleport user.  If you are experiencing long lookup times on Windows, do the following:

- Make sure `tsh` is at least at version 12.2.4 or upgrade to the latest version.
- Set `TELEPORT_USER` environment variable or set the `--user` flag to the name of your Teleport user.
- Set `TELEPORT_LOGIN` environment variable or set the `--login` flag to the name of current
host user. This should get overridden if you want to SSH to a machine as a different user.
- Set
`TELEPORT_HOME` environment variable to be the home directory of your current host user + `\.tsh`.
So if your home directory is `C:\Users\Me`, you'd set `TELEPORT_HOME` to `C:\Users\Me\.tsh`.

Reply:
Done